packages/bpk-component-barchart/src/BpkBarchartBars.js

```diff
@@ -42,7 +44,31 @@ const getBarHeight = (
 const isOutlier = (point, { yScaleDataKey, maxYValue }) =>
   point[yScaleDataKey] > maxYValue;
 
-const BpkBarchartBars = props => {
+type Props = {
+  data: Array<any>,
+  xScaleDataKey: string,
+  yScaleDataKey: string,
+  height: number,
+  xScale: Function,
+  yScale: Function,
+  maxYValue: number,
+  margin: {
+    top: number,
+    bottom: number,
+    left: number,
+    right: number,
+  },
+  getBarLabel: Function,
+  BarComponent: Function,
+  getBarSelection: Function,
+  outerPadding: number,
+  innerPadding: number,
+  onBarClick: ?(any?: any, any?: any) => mixed,
+  onBarHover: ?(any?: any, any?: any) => mixed,
+  onBarFocus: ?(any?: any, any?: any) => mixed,
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here?